src/components/Dropdown/DropdownMenu.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface DropdownMenuProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+/**
+ * A menu that is only visible when its `Dropdown` parent has an `open` attribute.
+ */
+export const DropdownMenu = ({
```

Comment:
I think this could be inlined, too, since it's only used in a single place!

Reply:
Good idea! Done.